src/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>TO Do List</title>
+</head>
+<body>
+  <main>
+    <div class="toDoItem">
+  <h1>Today's To Do</h1>
+  <div class="refresh">	&#x27F2;</div>
+</div>
+  <form id="addToDo"  class="toDoItem">
+    <input name="addList" type="text" id="addList" placeholder="Add to your list..." required>
+    <button type="submit" id="enter">	&#x2936;</button>
```

Comment:
Kindly consider using the aria-label attribute when using input tag. The aria-label attribute provides a text alternative for screen reader users to understand the purpose of the input element.